packages/mds-db/migration.ts

```diff
@@ -130,11 +134,24 @@ async function doMigration(exec: SqlExecuterFunction, migration: MIGRATION, migr
   }
 }
 
+async function alterAuditEventsColumns(exec: SqlExecuterFunction) {
```

Comment:
We are establishing a convention in one of Jane's PRs. Name this function `alterAuditEventsColumnsMigration` to make it more clear that you would not normally call this function outside of a migration.